components/composer/src/Composer.wc.svelte

```diff
@@ -1,11 +1,12 @@
 <svelte:options tag="nylas-composer" immutable />
 
 <script lang="ts">
-  import "./components/HTMLEditor.svelte";
-  import "./components/AlertBar.svelte";
-  import "./components/Attachment.svelte";
-  import "./components/DatepickerModal.svelte";
-  import "./components/ContactsSearch.svelte";
+  import HTMLEditor from "./components/HTMLEditor.svelte";
+  import AlertBar from "./components/AlertBar.svelte";
+  import ComposerAttachment from "./components/Attachment.svelte";
+  import DatepickerModal from "./components/DatepickerModal.svelte";
+  import ContactsSearch from "./components/ContactsSearch.svelte";
+  import NylasError from "@commons/components/DefaultError.svelte";
```

Comment:
Wouldn't this be `import DefaultError from "@commons/components/DefaultError.svelte";` or `import { DefaultError as NylasError } from "@commons/components/DefaultError.svelte";`?

Reply:
We could technically name the default export (`import DefaultError from "@commons/components/DefaultError.svelte";`) anything we'd like but using `NylasError` for consistency with the rest of the components. This is temporary until all components support child svelte components.